.jenkins/pytorch/macos-test.sh

```diff
@@ -108,14 +108,14 @@ test_custom_script_ops() {
 }
 
 
-if [ -z "${JOB_BASE_NAME}" ] || [[ "${JOB_BASE_NAME}" == *-test ]]; then
+if [ -z "${BUILD_ENVIRONMENT}" ] || [[ "${BUILD_ENVIRONMENT}" == *-test ]]; then
   test_python_all
   test_libtorch
   test_custom_script_ops
 else
-  if [[ "${JOB_BASE_NAME}" == *-test1 ]]; then
+  if [[ "${BUILD_ENVIRONMENT}" == *-test1 ]]; then
```

Comment:
This looks like dead code to me. @yf225 did there used to be more mac test jobs?

Reply:
We used to split the test job in half for parallel testing which is what this was for. We can probably keep them around in case we are doing it in the future (which would be a pretty nice CI speed improvement)